src/components/output/Output.vue

```diff
@@ -15,12 +16,17 @@ export default {
   components: { MessageOutput },
   props: {
     id: String,
+    preview: Boolean,
     alpha: Boolean,
-    showBackground: Boolean
+    showBackground: Boolean,
```

Comment:
Deze zou toch weg kunnen? Kan je id voor gebruiken.
Wanneer id = beamer --> tonen en ander niet.

Reply:
Dat zou kunnen, maar dan maak je het component niet "reusable", aangezien je het component dan zelf moet aanpassen voordat je het op een andere plek kan gebruiken. Over het algemeen is het goed als components/classes niks van de aanroeper afweten.